pkg/util/util.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+func StringSliceContains(slice []string, str string) bool {
```

Comment:
Should be an easy unit test.

Reply:
Done.

Reply:
Indeed, please consider the latest changes.